docs/clients/s3.md

```diff
@@ -67,3 +67,23 @@ This information is required by AWS, and cannot be guessed by AsyncAws.
 If `ContentLength` is absent, AsyncAws will read the output before sending the
 request which could have a performance impact.
 
+
+### Download files
+```
+$getObjectResult = $s3->GetObject([
+    'Bucket' => 'my-company-website',
+    'Key' => 'humans.txt',
+]);
+$fp = fopen('/path/to/file.txt', 'wb');
+fwrite($fp, $result->getBody()->getContentAsString());
```

Comment:
The above example is about dumping content in a file. Maybe we can leverage `getContentAsString` to perform business logic with that content

ie.
```php
$getObjectResult = $s3->GetObject([
    'Bucket' => 'my-company-website',
    'Key' => 'buildMetadata.json',
]);
$metadata = json_decode($result->getBody()->getContentAsString());
```